.github/container/Dockerfile.grok

```diff
@@ -1,24 +1,22 @@
 # syntax=docker/dockerfile:1-labs
 
 ARG BASE_IMAGE=ghcr.io/nvidia/jax-mealkit:jax
-ARG SRC_PATH_GROK=/opt/grok-1
+ARG URLREF_GROK_1=https://github.com/xai-org/grok-1.git#main
+ARG SRC_PATH_GROK_1=/opt/grok-1
```

Comment:
Why `_1`?

Reply:
This is the Grok-1 model and there may be future models named Grok-2, Grok-3, etc... 🤣